components/providers/ContentAccordion.vue

```diff
@@ -1,53 +1,40 @@
 <template>
-  <cv-accordion ref="acc" class="content-accordion" @change="actionChange">
+  <bx-accordion ref="acc" class="content-accordion" @change="actionChange">
     <div
       v-for="(tab, index) in tabs"
       :key="tab.title"
-      @click.capture="preventFromClosing(index, $event)"
     >
-      <cv-accordion-item
-        :key="tab.title"
+      <bx-accordion-item
         :open="index === expandedItem"
+        :title-text="tab.title"
       >
-        <template slot="title">
-          {{ tab.title }}
-        </template>
-        <template slot="content">
-          <AccordionLayout
-            :image="tab.content.image"
-            :description="tab.content.description"
-            :cta="tab.content.cta"
-          />
-        </template>
-      </cv-accordion-item>
+        <AccordionLayout
+          :title="tab.title"
+          :description="tab.description"
+          :installation="tab.installation"
+          :image="tab.image"
+          :website-cta="tab.websiteCta"
+          :docs-cta="tab.docsCta"
+          :source-cta="tab.sourceCta"
+          :hello-world-example="tab.helloWorldExample"
```

Comment:
Not sure if `hello-world-example` is a good name... I think just `code-example` defines it better, what do you think?

Reply:
we can change to `code-example` and the same in the json file.

Reply:
done here, in the component props. 

However, I will wait for @HuangJunye to update the JSON property, then I can make the full change on our side in the `providersContent.ts` > `type ProviderObject`